bin/dependency-check-pr.sh

```diff
@@ -22,22 +25,21 @@ main() {
 
     local CURRENT_TAG
     CURRENT_TAG="$(yq ".dependencies.${NAME}.current_tag" "${DEPENDENCIES_YML}")"
-    echo "Current Tag: ${CURRENT_TAG}"
+    echo -e "Current Tag: ${white}${CURRENT_TAG}${white}"
```

Comment:
```suggestion
    echo -e "Current Tag: ${white}${CURRENT_TAG}${reset}"
```
should this be reset at the end?